llvm.cpp

```diff
@@ -20,8 +20,7 @@ class Cowboy {
   int Age;
 };
 
-Cowboy::Cowboy()
-    : Age(45), GunCount(2) {
```

Comment:
i think i found the change [here](https://llvm.org/docs/CodingStandards.html#anonymous-namespaces)